locations/spiders/aramark_uniform_services_us.py

```diff
@@ -59,10 +60,20 @@ def parse(self, response):
             item.update(extra_fields_1)
             extra_fields_2 = {k: v for k, v in DictParser.parse(location["node"]["postLocation"]).items() if v}
             item.update(extra_fields_2)
-            if "Uniform Services" in item["name"].title():
-                item["brand"] = "Aramark Uniform Services"
+
+            def add_tags(item):
+                apply_category(Categories.SHOP_LAUNDRY, item)
+                apply_yes_no("laundry_service", item, True)
+                apply_category({"rental": "clothes"}, item)
+                apply_category({"access": "private"}, item)
+
+            if "Cleanroom Services" in item["name"].title():
+                item["brand"] = "Vestis Cleanroom Services"
                 item["name"] = item["brand"] + " " + item["city"]
```

Comment:
I know this was there already, but we shouldn't craft our own name for a place unless this is what the brand themselves use. Even then, I would prefer to pull it from the page rather than build it ourselves by concatenating strings.

Reply:
Understandable. I did look if they have anything and the location (`location["node"]["postLocation"]["name"]`) only have 2 outputs, `Vestis` and `Vestis Cleanroom Services`. We could change it to just use that.